[DOCS] resource_name property for attachment ingest processor (#65974)
elastic/elasticsearch: docs/plugins/ingest-attachment.asciidoc

| `indexed_chars_field`  | no        | `null`           | Field name from which you can overwrite the number of chars being used for extraction. See `indexed_chars`.
| `properties`           | no        | all properties   | Array of properties to select to be stored. Can be `content`, `title`, `name`, `author`, `keywords`, `date`, `content_type`, `content_length`, `language`
| `ignore_missing`       | no        | `false`          | If `true` and `field` does not exist, the processor quietly exits without modifying the document

> | `resource_name`        | no        |                  | The field containing the name of the resource to be decoded. If supplied, this is provided to
>                                                           the underlying Tika library to enable https://tika.apache.org/1.24.1/detection.html#Resource_Name_Based_Detection[Resource Name Based Detection].

Reviewer: Tried to streamline slightly. The current copy is fine as-is though.
```suggestion
| `resource_name`        | no        |                  | Field containing the name of the resource to decode. If specified, the processor passes this resource name to the underlying Tika library to enable https://tika.apache.org/1.24.1/detection.html#Resource_Name_Based_Detection[Resource Name Based Detection].
```
